decoders/avr_isp/parts.py

```diff
@@ -22,20 +22,76 @@
 
 # Vendor code
 vendor_code = {
-    0x1e: 'Atmel',
+    0x1E: 'Atmel',
```

Comment:
unrelated cosmetic change, in the future put this in separate commit (with all other similar fixes).